policyengine_us/variables/gov/ed/pell_grant/sai/eligibility_type/pell_grant_min_fpg_limit.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class pell_grant_min_fpg_percent_limit(Variable):
+    value_type = float
+    entity = Person
+    definition_period = YEAR
+    label = "The maximum FPG percent to qualify for the minimum Pell Grant"
+
+    def formula(person, period, parameters):
+        dependent = person("is_tax_unit_dependent", period)
+        joint = person.tax_unit("tax_unit_is_joint", period)
+        not_parent = person.tax_unit("tax_unit_child_dependents", period) == 0
+
+        p = parameters(period).gov.ed.pell_grant.sai.min_pell_limits
+
+        return select(
+            [
+                dependent & ~joint,
+                dependent & joint,
+                ~dependent & not_parent,
+                ~dependent & ~joint,
+                ~dependent & joint,
+            ],
+            [
+                p.DEPENDENT_SINGLE,
```

Comment:
I think it might be cleaner to create an ENUM variable for the five cases and the corresponding conditions - we can then use it in the max and min variables and avoid duplicate select statements

Reply:
There are slight differences in the logic. I don't account for tax filing because I could not find a variable for it. I do account for whether or not the person is a parent, and that is a difference between the min and max Pell Grant.

# Maximum
![image](https://github.com/PolicyEngine/policyengine-us/assets/62856626/a642d7c7-f9c7-4440-8deb-c7115fcbc3a8)
# Minimum
![image](https://github.com/PolicyEngine/policyengine-us/assets/62856626/87061e3f-aeba-4479-94d4-970c40063916)